### PR TITLE
Simplify vs2015 build

### PR DESCRIPTION
**Pull request by dbc60**
planning on using jsoncpp as a submodule in tsEventQuery. There's no need to copy debug/release builds of lib_json.lib to a single location, nor is there a need to call the debug version lib_jsond.lib.

### CODE REVIEW

**Comment by yankeehacker** [comment]
LGTM

File: makefiles/msvc2015/lib_json.vcxproj [hunk 1]
```diff
@@ -108,10 +105,7 @@
       <SubSystem>Windows</SubSystem>
       <GenerateDebugInformation>true</GenerateDebugInformation>
     </Link>
-    <PostBuildEvent>
-      <Command>xcopy /f /y $(OutputPath)$(TargetFileName) $(SolutionDir)..\..\lib\
```
Comment (michaelchen16) [inline makefiles/msvc2015/lib_json.vcxproj]: What did this actually do?

Comment (dbc60) [inline makefiles/msvc2015/lib_json.vcxproj]: After the build it copied the lib from the Debug/Release directory to a common location. The debug version was called lib_jsond.lib so it wouldn't clash with the release version. At the time it seemed to be an easier way to link it into tsEventQuery.exe <sigh>.

Comment (michaelchen16) [inline makefiles/msvc2015/lib_json.vcxproj]: Ah, okay!

**Comment by michaelchen16** [comment]
LGTM!

**Pull request merged**